android/src/main/AndroidManifest.xml

```diff
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would not delete this file. I would instead remove the line from the example AndroidManifest that specifies the service:
```xml
<service
            android:name="com.guichaguri.trackplayer.service.MusicService"
            android:enabled="true" />
```

Plus we need the `<receiver android:name="androidx.media.session.MediaButtonReceiver">` to be registered.

Either way I think its best that the lib has this than asking the apps to add this to their Manifest files. Most people are generating RN projects via CLI and they shouldn't have to modify this to make the player work.